packages/block-editor/src/components/block-list/style.scss

```diff
@@ -386,8 +386,9 @@
 	cursor: text;
 
 	// Hide the inserter above the selected block.
-	&.is-inserter-hidden {
+	&.is-inserter-hidden .block-editor-inserter__toggle {
 		opacity: 0;
+		pointer-events: none;
```

Comment:
I've noticed one negative impact of this style: When trying to scroll the block list, if the cursor happens to be placed over the sibling inserter when using the scroll wheel, the scroll will not happen.

Reply:
Hm, that's not good. :)

Reply:
Hm, it seems to be happening before this change as well.

Reply:
The problem is that the pointer is over a `Popover`, which won't scroll the page.

Reply:
I notice the same thing with the block toolbar. It's not possible to scroll with the pointer over it.

Reply:
Hm, I would suppose that's because the popover is in another place in the DOM, not part of the scrollable region. Not sure how we'd solve that, other than forcing the Popover to be rendered in the same DOM tree, reverse those refactorings to not use popover, or somehow "replay" that scrolling into the scroll container. Maybe there's some option for ignoring scroll events that happen in the popover (something like taking advantage of what I suspected originally to be the issue, via `pointer-events`, maybe dynamically applied?).

Reply:
The strange thing is that it _is_ rendered in the scroll container.

Reply:
Is there anything in the popover rendering that would be preventing the scroll? `pointer-events` or `event.preventDefault` ?

Reply:
It's not immediately obvious to me. I commented out all pointer-events rules and it's still not scrolling. Also no scroll preventing I think.

Reply:
>The strange thing is that it is rendered in the scroll container.

In some further debugging:

- It _is_ part of the container where scrolling is applied (`.edit-post-editor-regions__content`)
- But it's _not_ part of the content which is causing the scroll (`edit-post-visual-editor`)

It's not immediately clear to me what the expected behavior here would be, but this could explain why the scroll doesn't happen.

![image](https://user-images.githubusercontent.com/1779930/72752617-aa7efd80-3b90-11ea-8cdb-24a78b247ce1.png)

Reply:
I found the issue. The popover container's position is fixed which means it's positioned relative to the viewport. Relatively positioning the scroll container won't help unless we absolutely position the popover. It sounds like this will need some adjustment in the `Popover` component.